addons/project/migrations/12.0.1.1/openupgrade_analysis.txt

```diff
@@ -89,7 +89,7 @@ NEW mail.template: project.rating_project_request_email_template (noupdate)
 DEL mail.template: project.mail_template_data_module_install_project (noupdate)
 DEL mail.template: rating_project.rating_project_request_email_template
 DEL project.task: project.project_task_data_8 (noupdate)
-DEL res.company: base.main_company (noupdate)
+DEL res.company: base.main_company [moved to web module] (noupdate)
```

Comment:
This is still strange that appears as this is not true. What about the approach I told about that ignoring records from other modules that are re-declared (for changing any field) in other modules?

Reply:
The ignoring is done for the `rename` check. If I do for the `move` check, then the moved groups from `sale` to `account` cannot be detected.